test/kwtool_web/controllers/api/v1/search_controller_test.exs

```diff
@@ -0,0 +1,220 @@
+defmodule KwtoolWeb.Api.V1.SearchControllerTest do
+  use KwtoolWeb.ConnCase, async: true
+
+  import KwtoolWeb.Support.ConnHelper
+
+  describe "get index/2" do
+    test "given search with keyword param, returns a list of matching keyword", %{
```

Comment:
```suggestion
    test "given search with keyword param, returns a list of matching keywords", %{
```

Reply:
Updated in https://github.com/byhbt/kwtool/pull/49/commits/634ec9ecd477dff893d4f82ebe2a8ab75e064701